config/locales/sk.yml

```diff
@@ -340,18 +341,20 @@ sk:
         expired: Vypršalo
         title: Filtruj
       title: Pozvánky
+          pending_accounts:
```

Comment:
This line is incorrectly indented, it should be at the same level as `relays:` a few lines later!